locale/en-US/shared.properties

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## LOCALIZATION NOTE: In this file, don't translate the part between {{..}}
+
+# Text chat strings
+chat_textbox_placeholder=Type here…
+
+## LOCALIZATION NOTE(clientShortname2): This should not be localized and
+## should remain "Firefox Hello" for all locales.
+clientShortname2=Firefox Hello
+
+conversation_has_ended=Your conversation has ended.
+generic_failure_message=We're having technical difficulties…
+
+generic_failure_no_reason2=Would you like to try again?
+
+help_label=Help
+
+mute_local_audio_button_title=Mute your audio
+mute_local_video_button_title2=Disable video
+
+## LOCALIZATION NOTE (retry_call_button):
+## This button is displayed when a call has failed.
+retry_call_button=Retry
+
+rooms_leave_button_label=Leave
+
+rooms_panel_title=Choose a conversation or start a new one
+
+rooms_room_full_call_to_action_label=Learn more about {{clientShortname}} »
+rooms_room_full_call_to_action_nonFx_label=Download {{brandShortname}} to start your own
+rooms_room_full_label=There are already two people in this conversation.
+rooms_room_join_label=Join the conversation
+rooms_room_joined_label=Someone has joined the conversation!
+
+## LOCALIZATION NOTE (tos_failure_message): Don't translate {{clientShortname2}}
```

Comment:
As you're here, please fix the note as it should reference clientShortname (it gets replaced by clientShortname2).